massa-pos-exports/src/deferred_credits.rs

```diff
@@ -70,7 +167,7 @@ impl DeferredCredits {
 
     /// Insert/overwrite a deferred credit
     pub fn insert(&mut self, addr: Address, slot: Slot, amount: Amount) {
-        let entry = self.0.entry(slot).or_default();
+        let entry = self.credits.entry(slot).or_default();
         entry.insert(addr, amount);
```

Comment:
This insertion shouldn't have an extend ?

Reply:
Extend? You mean hash update? If yes this is also only used speculatively, no need to do so